app/dashboard/utils.py

```diff
@@ -561,3 +559,13 @@ def test(addrstr):
     # return priv key, pub key, address
 
     return priv.to_string().hex(), pub.hex(), checksum_encode(address)
+
+
+def get_bounty_semaphore_ns(standard_bounty_id):
+    return f'{SEMAPHORE_BOUNTY_NS}_{standard_bounty_id}_{SEMAPHORE_BOUNTY_SALT}'
+
+
+def release_bounty_lock(standard_bounty_id):
```

Comment:
is this being called anywhere?

Reply:
No - merely introduced it as a mechanism for us to manually lift a lock in the event we needed to.  @owocki

Reply:
makes sense